test/extended/util/image/image.go

```diff
@@ -27,7 +27,10 @@ func init() {
 
 		// used by build s2i e2e's to verify that builder with USER root are not allowed
 		// the github.com/openshift/build-test-images repo is built out of github.com/openshift/release
-		"registry.ci.openshift.org/ocp/4.7:test-build-roots2i": -1,
+		"registry.ci.openshift.org/ocp/4.8:test-build-roots2i": -1,
+
+		// used by all the rest build s2s e2e tests
+		"registry.ci.openshift.org/ocp/4.8:test-build-simples2i": -1,
```

Comment:
OK let's temporarily remove this @alicerum so we can prove that the `USER 1001` fix to registry.ci.openshift.org/ocp/4.8:test-build-simples2i worked.

registry.ci.openshift.org/ocp/4.8:test-build-simples2i has been updated with your recently merged build-test-images PR